mapper/mapper.go

```diff
@@ -69,6 +69,17 @@ func CreateFilterableLandingPage(ctx context.Context, req *http.Request, d datas
 	p.DatasetId = datasetID
 	p.ReleaseDate = ver.ReleaseDate
 	p.BetaBannerEnabled = true
+	if d.Type == "nomis" {
+		log.Event(ctx, "delete me", log.WARN, log.Data{"datasetDetails": d, "datasetVersion": ver})
```

Comment:
If planning to keep the log , will it be better to have a different event name

Reply:
Haha, whoopsie